Lab-6.ipynb

```diff
@@ -0,0 +1,631 @@
+    "fig, (ax1,ax2) = plt.subplots(1,2, figsize=(18,8),gridspec_kw={'wspace': 0.4})\n",
+    "plt.style.use('ggplot')\n",
+    "\n",
+    "ax1.set_title(\"Signal Injected Data, Background $\\mu=100$\",fontsize=20)\n",
+    "ax2.set_title(\"Log Signal Injected Data, Background $\\mu=100$\",fontsize=20)\n",
+    "ax1.set_xlabel('Observed Events',fontsize=20), ax2.set_xlabel('Observed Events',fontsize=20)\n",
+    "ax1.set_ylabel('Counts',fontsize=20), ax2.set_ylabel('Log(Probability)',fontsize=20)\n",
+    "ax1.tick_params(labelsize=14), ax2.tick_params(labelsize=14)\n",
+    "\n",
+    "ax1.hist(inject,color='b',bins=400,label='Signal injection')\n",
+    "ax2.hist(inject,color='b',density=True,bins=100,label='Signal injection')\n",
+    "ax2.set_yscale('log')\n",
+    "\n",
+    "ax1.legend(loc='best',fontsize=16)\n",
+    "ax2.legend(loc='upper right',fontsize=16)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "These histograms show us the background and the observed data distributions post-signal injection. The log plot shows more clearly the true distribution shapes and the minor changes along the curves; as we can see, both the background and the observation shapes appear parabolic on the log plot and therefore approached a Gaussian distribution.\n",
+    "\n",
+    "After signal injection into the background, there are two notable changes to the histogram distributions: One, the peaks have changed. Both curves have shorter peaks (in counts) than pre-signal injection. Second, most importantly, the observed signal has changed. While the background remains the same as expected, since our fake background doesn't change with time or any other expected changes, the signal histogram shape has now shifted up on the x-axis. Looking closer, we see the peak, where the mean is located, has shifted up exactly by 100, from 215 to 315.\n",
+    "\n",
+    "This upwards shift along the x-axis happens due to the additive nature of the background. The background mean is 100, therefore all values within the signal distribution have 100 added.\n",
+    "\n",
+    "What we are looking at then is the background and the signal together. The true signal is not actually this much stronger. Instead, the background is just adding to the value we observe or measure. To get the true signal, we would have to subtract the background out in a real experiment."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The observed signal above is very symmetric about the mean=215 signal strength, as seen in both the histogram and log plots above. There is very little variation or change away from a symmetric Gaussian shape. Therefore, the observed signal that we measure is not biased. Since both the background and the signal follow the same type of distribution, we don't expect the observed signal value to stray far from the Gaussian."
```

Comment:
Even though we gave the same proof, I believe we came to a different conclusion on whether the observed signal is biased or not. I thought meant (is the observed value equally likely to be stronger or weaker than the true injected signal?). I thought it is bias because the observed signal is always greater than the true signal because the background is always positive.

Reply:
Good point. Thanks for fixing this, I misunderstood the bias part.

Reply:
I think you're right about this even though the background can add or subtract more or less, it will still be positive. Thanks!